src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/playback/GerritMissedEventsPlaybackManager.java

```diff
@@ -155,6 +158,38 @@ protected synchronized Date getDateFromTimestamp() {
         return new Date();
     }
 
+    @Override
+    public void run() {
```

Comment:
Probably better implemented as an [AsyncPeriodicWork](http://javadoc.jenkins.io/hudson/model/AsyncPeriodicWork.html) to save on thread count.